ocs_ci/utility/pagerduty.py

```diff
@@ -21,18 +21,23 @@ def set_pagerduty_integration_secret(integration_key):
         integration_key (str): Integration key taken from PagerDuty Prometheus integration
 
     """
-    logger.info("Setting up PagerSuty integration")
+    logger.info("Setting up PagerDuty integration")
     kubeconfig = os.getenv("KUBECONFIG")
+    addon_name = config.DEPLOYMENT["addon_name"]
     cmd = (
-        f"oc create secret generic {constants.MANAGED_PAGERDUTY_SECRET} "
+        f"oc create secret generic {addon_name + constants.MANAGED_PAGERDUTY_SECRET_SUFFIX} "
```

Comment:
We could define variable like `pagerduty_secret = addon_name + constants.MANAGED_PAGERDUTY_SECRET_SUFFIX` in the begining and then use `pagerduty_secret ` in the commands (it's used 3 times there).